icu4c/source/test/intltest/units_test.cpp

```diff
@@ -1205,6 +1205,7 @@ void UnitsTest::testUnitsConstantsDenomenator() {
         {"portion-per-7", 7},
         {"portion-per-8", 8},
         {"portion-per-9", 9},
```

Comment:
These cases don't exist in the Java tests.
Is there a good reason to have them?
Are they testing something interesting, that is not well covered by other cases?
They they should also be in Java.

If they don't actually exercise anything interesting, we should not have them in C/C++ either.

In general the tests in C++ and Java should be the same, unless there are very good reasons (for example Java testing for all kind of types that don't exist in C++)

Reply:
now , c++ and java are identical, sorry for this small confusion.